### DISCUSSION TITLE
Rpi Zero W compatible?

### DISCUSSION THREAD

**Discussion by ScriptGeek** [davixdedem/Pi-Starlink#6] [Q&A]
Will this work with a Raspberry Pi Zero W with 10/100 Ethernet/USB HUB hat?

**Comment by davixdedem** [comment]
Because I've never tried I can't be sure 100%, but why it shouldn't work?  I've just seen on the official OpenWRT Webpage that there's full compatibility. Have a look here: [RPi Zero W - OpenWRT](https://downloads.openwrt.org/releases/23.05.4/targets/bcm27xx/bcm2708/openwrt-23.05.4-bcm27xx-bcm2708-rpi-ext4-factory.img.gz)

**Reply by ScriptGeek** [reply]
Thanks, I'll give it a try. I've tried using RaspAP, but got stuck trying to get the ethernet port setup for DHCP to pull an IP from the Starlink dish. I think the ethernet USB hat complicates things, as none of the 400 something tutorials I found offered much help, just many dead-ends.
